extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -422,7 +422,11 @@ export class ProjectsController {
 
 			publishDatabaseDialog.openDialog();
 
-			return publishDatabaseDialog.waitForClose();
+			await publishDatabaseDialog.waitForClose();
+
+			if (publishDatabaseDialog.profileSaved) {
```

Comment:
an alternative is to call the dataworkspace.refresh command from the publishDatabaseDialog after the 'Save' button is clicked so the `profileSaved` property wouldn't be needed

Reply:
Done.